src/main/java/com/bookstore/bookstoreapi/data/controller/BookController.java

```diff
@@ -0,0 +1,26 @@
+package com.bookstore.bookstoreapi.data.controller;
+
+import com.bookstore.bookstoreapi.data.entity.Book;
+import com.bookstore.bookstoreapi.data.service.BookService;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class BookController {
+
+  @Autowired private BookService bookService;
+
+  @PostMapping("/books")
```

Comment:
not a proper way to name this POST req, more appropriate would be "book/create" or add or something similar.